rbac/management/audit_log/model.py

```diff
@@ -138,3 +153,17 @@ def log_edit(self, request, resource, object):
         self.action = AuditLog.EDIT
         self.tenant_id = self.get_tenant_id(request)
         super(AuditLog, self).save()
+
+    def log_group_assignment(self, request, resource, object, type_dict, user_type):
```

Comment:
`user_type` could be user, service or role.  What about name `target_resource_type` or `assigned_resource_type` ? or something like that